vm/ieleApi.ml

```diff
@@ -27,15 +27,14 @@ let get_block blocknumber =
 
 let personal_newAccount () =
   let acct = Cryptokit.Random.string random 20 in
-  let key = to_hex (Bytes.of_string acct) in
+ `String (to_hex (Bytes.of_string acct))
+
+let setBalance (address, account) =
   let value = `Assoc [("nonce", `String "0x00"); ("balance", `String "0x00"); ("storage", `Assoc []); ("code", `String "")] in
   let latest = List.hd !blocks in
   let state = latest.state in
-  let new_state = (key, value) :: state in
+  let new_state = (address, value) :: state in
```

Comment:
What is setBalance supposed to do if the address already exists in the state?

Reply:
Note that this is called by setChainParams which first wipes the entire state, so that can't happen.

Reply:
Ah, it's not exported, then just documenting the precondition is fine - `(* assumes the account has no entry in state *)` or something.